### DISCUSSION TITLE
Evolution E_Int -> Int and E_Float -> Float

### DISCUSSION THREAD

**Discussion by benoit128** [onera/Cassiopee#98] [Ideas]
Hello,

Pour faciliter la lecture et la simplicité, on propose de remplacer partout dans le code C :
E_Int par Int
E_Float par Float

Pros : plus rapide à taper, plus facile à lire
Cons: confustion possible entre int et Int.

Merci de donner vos avis!

**Comment by vincentcasseau** [comment]
Il y aurait aussi: `E_Bool` -> `Bool`

**Comment by nalferez** [comment]
Bonjour,

J'imagine que mon avis est secondaire. Cependant, je pense que la notation E_Int est aussi plus avantageuse pour une recherche globale à travers les sources ("Int" peux plus facilement se retrouver dans une chaine de caractère, p.e. "Intersector").

Nicolas

Envoyé à partir de Outlook pour Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by benoit128** [comment]
Salut NIcoolas,
Ton avis n'est pas secondaire, merci de participer à la discussion!
a+

**Comment by benoit128** [comment]
Après discussions, on reste sur E_Int et E_Float pour l'instant.
